components/src/core/Drawer/DrawerNavItem.tsx

```diff
@@ -134,6 +137,13 @@ const useStyles = makeStyles((theme: Theme) =>
             paddingRight: (props: DrawerNavItemProps): string =>
                 theme.direction === 'ltr' ? theme.spacing(2) : calcNestedPadding(theme, props.depth),
         },
+        listItemButtonRoot: {
```

Comment:
rather than defining the exact same rules twice, could we just have the one rule (`infoListItemRoot`) and conditionally apply it to either the InfoListItem or the ListItemButton as appropriate?

Reply:
I have updated code and using shared class which will apply as per condition.